modules/Bio/EnsEMBL/Compara/PipeConfig/EPOwith2x_conf.pm

```diff
@@ -0,0 +1,302 @@
+        'enredo_output_file'    => $self->o('enredo_output_file'),
+        'bed_dir'               => $self->o('bed_dir'),
+        'genome_dumps_dir'      => $self->o('genome_dumps_dir'),
+        'enredo_mapping_file'   => $self->o('enredo_mapping_file'),
+        'bl2seq_dump_dir'       => $self->o('bl2seq_dump_dir'),
+        'bl2seq_file_stem'      => $self->o('bl2seq_file_stem'),
+
+        # databases
+        'compara_anchor_db' => $self->o('compara_anchor_db'),
+        'master_db'         => $self->o('master_db'),
+        'reuse_db'          => $self->o('reuse_db'),
+        'ancestral_db'      => $self->o('ancestral_db'),
+        'epo_db'            => $self->o('epo_db'),
+
+        # options
+        'run_gerp' => $self->o('run_gerp'),
+    };
+
+}
+
+sub core_pipeline_analyses {
+    my ($self) = @_;
+
+    return [
+
+        {   -logic_name => 'start_prepare_databases',
+            -module     => 'Bio::EnsEMBL::Hive::RunnableDB::Dummy',
+            -input_ids  => [{
+                mlss_id => '#high_epo_mlss_id#',
+            }],
+            -flow_into  => {
+                '1->A' => [ 'copy_table_factory', 'set_internal_ids', 'drop_ancestral_db' ],
+                'A->1' => 'reuse_anchor_align_factory',
+            }
+        },
+        
+        {   -logic_name => 'mlss_factory',
+            -module         => 'Bio::EnsEMBL::Hive::RunnableDB::JobFactory',
+            -parameters => {
+                'inputlist'    => [ 
+                    [
+                        '#high_epo_mlss_id#', 
+                        ['EPO'], 
+                        1, 
+                        ['high_epo_mlss_id']
+                    ],
+                    [
+                        '#low_epo_mlss_id#', 
+                        ['EPO_LOW_COVERAGE', 'GERP_CONSTRAINED_ELEMENT', 'GERP_CONSERVATION_SCORE'], 
+                        0,
+                        ['low_epo_mlss_id', 'ce_mlss_id', 'cs_mlss_id'],
+                    ],
+                ],
+                'column_names' => [ 'mlss_id', 'whole_method_links', 'is_good_for_alignment', 'param_names' ],
+            },
+            -flow_into  => {
+                '2->A' => 'create_mlss_ss',
+                'A->1' => 'set_gerp_mlss_tag',
+            }
+        },
+        
+        {   -logic_name => 'setup_low_coverage_alignment',
+            -module     => 'Bio::EnsEMBL::Hive::RunnableDB::Dummy',
+            -flow_into  => {
+                '1->A' => 'create_default_pairwise_mlss',
+                'A->1' => 'dump_mappings_to_file',
+            }
+        },    
+        
+        {   -logic_name => 'set_internal_ids_low_epo',
+            -module => 'Bio::EnsEMBL::Compara::RunnableDB::PairAligner::SetInternalIdsCollection',
+            -parameters => {
+                method_link_species_set_id => '#low_epo_mlss_id#',
+            },
+            -flow_into => [ 'create_neighbour_nodes_jobs_alignment' ],           
```

Comment:
You declare this but then in line 266 you have:
```
$analyses_by_name->{'set_internal_ids_low_epo'}->{'-flow_into'} = ['update_max_alignment_length'];
```
I guess you can declare that flow here and remove that tweak?